tests/data/index.js

```diff
@@ -42,10 +42,21 @@ import SequencescapeLabwareTubeOnly from './sequencescapeLabwareTubeOnly'
 import SequencescapeLabwarePlateOnly from './sequencescapeLabwarePlateOnly'
```

Comment:
This is starting to look quite hairy. Seems like we need a different method for getting data. Something to ponder.

Reply:
Yeah i think it would be good at some point to make a factory that could also generate responses so we can combine cypress fixtures and unit tests